fail2ban/server/filterpyinotify.py

```diff
@@ -113,14 +151,81 @@ def _process_file(self, path):
 			self.failManager.cleanup(MyTime.time())
 		self.__modified = False
 
+	def _addPending(self, path, reason, isDir=False):
+		if path not in self.__pending:
+			self.__pending[path] = [self.sleeptime / 10, isDir];
```

Comment:
Why by 10? Now that in some obscure case could become 0, so much be worth ceiling float division instead